src/i18n/translations/ar.js

```diff
@@ -0,0 +1,150 @@
+const de = {
```

Comment:
This needs to be named the same as the code:

```suggestion
const ar = {
```